tables/rx_potency/yuan22-pot.csv

```diff
@@ -127,6 +127,7 @@ Yuan22,REGN10987,BA.1 Spike,Figure 2A,Pseudovirus (MLV),IC50,5000,1,5000,1,ng/ml
 Yuan22,AZD1061+AZD8895,BA.1 Spike,Figure 2A,Pseudovirus (MLV),IC50,1313,1,5000,1,ng/ml,3/17/22
 Yuan22,REGN10987+REGN10933,BA.1 Spike,Figure 2A,Pseudovirus (MLV),IC50,5000,1,5000,1,ng/ml,3/17/22
 Yuan22,ADG20,BA.1 Spike,Figure 2A,Pseudovirus (MLV),IC50,1203,1,5000,1,ng/ml,3/17/22

```

Comment:
Please update the date format

Reply:
Did it need to be the current date, or same date as before?

Reply:
Same dates are ok, since you haven't changed them.